openapi/workflow_execution_service.openapi.yaml

```diff
@@ -596,11 +675,19 @@ components:
           $ref: '#/components/schemas/State'
         run_log:
           $ref: '#/components/schemas/Log'
+        task_logs_url:
+          type: string
+          description: A reference to the complete url which may be used to obtain a paginated list of task logs for this workflow
         task_logs:
           type: array
+          deprecated: true
+          required: false
           items:
             $ref: '#/components/schemas/Log'
-          description: The logs, and other key info like timing and exit code, for each step in the workflow run.
+          description:  >-
+              The logs, and other key info like timing and exit code, for each step in the workflow run.
+              This field is deprecated and the `task_logs_url` should be used to retrieve a paginated list
+              of steps from the workflow run
```

Comment:
Perhaps we should be specific as to when the field will be removed, e.g., for the next major version release `2.0.0`?